tools/pd-ctl/pdctl/command/health_command.go

```diff
@@ -15,30 +15,25 @@
 package command
 
 import (
-	"net/http"
-
 	"github.com/spf13/cobra"
 )
 
-var (
-	healthPrefix = "pd/api/v1/health"
-)
-
 // NewHealthCommand return a health subcommand of rootCmd
 func NewHealthCommand() *cobra.Command {
 	m := &cobra.Command{
-		Use:   "health",
-		Short: "show all node's health information of the pd cluster",
-		Run:   showHealthCommandFunc,
+		Use:               "health",
+		Short:             "show all node's health information of the PD cluster",
+		PersistentPreRunE: requirePDClient,
```

Comment:
`TestSendAndGetComponent` ci failed :(

- This is because the `requirePDClient` command executes the function `shouldInitPDClient`, which requires querying `pd/api/v1/cluster`.

- The handler is replaced in `TestSendAndGetComponent`, so you need to add this router to the handler in `TestSendAndGetComponent`.

```
		mux.HandleFunc("/pd/api/v1/cluster", func(w http.ResponseWriter, r *http.Request) {
			cluster := &metapb.Cluster{
				Id: 0,
			}
			var rd render.Render
			rd.JSON(w, http.StatusOK, cluster)
		})
```

Reply:
You can check code here

https://github.com/tikv/pd/blob/7e18a69734f2ae391516625ec0230c05b363f5c5/tools/pd-ctl/tests/global_test.go#L37-L52

Reply:
Thanks for your reminder and suggestion. I will fix it tomorrow.